client/gutenberg/extensions/subscriptions/edit.jsx

```diff
@@ -3,47 +3,96 @@
 /**
  * External dependencies
  */
-import { Component } from '@wordpress/element';
-import { TextControl, Button, ToggleControl } from '@wordpress/components';
+import { Component, Fragment } from '@wordpress/element';
+import { TextControl, Button, ToggleControl, Disabled } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 import { sprintf } from '@wordpress/i18n/build/index';
 import apiFetch from '@wordpress/api-fetch';
+import {
+	URLInput,
+	RichText,
+	ContrastChecker,
+	InspectorControls,
+	withColors,
+	PanelColorSettings,
+} from '@wordpress/editor';
 
 class SubscriptionEdit extends Component {
 	render() {
 		const { attributes, className, isSelected, setAttributes } = this.props;
-		const { subscribe_placeholder, show_subscribers_total, subscriber_count_string } = attributes;
+		const {
+			subscribe_placeholder,
+			show_subscribers_total,
+			subscriber_count_string,
+			button_background_color,
+			button_text_color,
+			button_large,
+		} = attributes;
 
+		const buttonStyle = {
+			'background-color': button_background_color,
+			color: button_text_color,
+			border: 'none',
```

Comment:
Are users expected to be able to modify `border` in future? If not, it should prolly be in CSS?

Reply:
The button block and Michael's designs account for a flat design, so I'm not accounting for border colours to be editable.